src/daily_output.ts

```diff
@@ -335,6 +342,25 @@ export class DailyOutput {
 		this.data.blockHeight = currentHeight;
 	};
 
+	private doublesData(txDate: Date, appName?: string, appPlatform?: string): boolean {
+		const MOBILE_x2_START_DATE = new Date('10/30/2022');
+		const MOBILE_x2_END_DATE = new Date('11/27/2022');
+		const daysDiffToStart = daysDiffInEST(txDate, MOBILE_x2_START_DATE);
+		const daysDiffToEnd = daysDiffInEST(txDate, MOBILE_x2_END_DATE);
+		const doublingAlreadyStarted = daysDiffToStart === 0;
+		const doublingDidntFinish = daysDiffToEnd <= 7 * 4;
```

Comment:
Works just perfectly for the dates BEFORE and WHILE happening, but AFTER the event it's still doubling the mobile uploads' data count. 

TODO: better determine when the event finishes

Reply:
[fc6d906](https://github.com/ardriveapp/inferno-backend/pull/56/commits/fc6d906e9c2762d9bf103c8de848e17e20f79194)